src/to_number.cc

```diff
@@ -0,0 +1,99 @@
+/**
+ * \file    to_number.cc
+ * \brief   Implementation of pow10().
+ * \authors \ref contributors
+ * \date    Created on 19 Februar 2024
+ *
+ * \copyright Copyright 2024 Deutsches Elektronen-Synchrotron (DESY), Hamburg
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 2.1 of the license, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include <float.h>
+
+#include "gul14/to_number.h"
+
+namespace gul14 {
+
+namespace detail {
+
+long double pow10(int exponent)
+{
+    // The preprocessor macros below just allow long double with
+    //
```

Comment:
Why the empty comment line?

Reply:
This separates the text from a list, changed it like this to make it more clear:

```
    // The preprocessor macros below just allow long double with
    //
    // - exponent maximum 511 (typical is 308) which is usually
    //   the case when long double == double (the code needs at least
    //   an allowed maximum of 256 for the constants)
    //
    // OR
    //
    // - exponent maximum 8191 (typical 4932) which is usually
    //   the case for 80 bit 'extended precision' or 'float128'
    //   long doubles (the code needs at least an allowed maximum
    //   of 4096 for the constants)
```